test/images/agnhost/agnhost.go

```diff
@@ -73,6 +74,7 @@ func main() {
 	rootCmd.AddCommand(fakegitserver.CmdFakeGitServer)
 	rootCmd.AddCommand(fakeregistryserver.CmdFakeRegistryServer)
 	rootCmd.AddCommand(guestbook.CmdGuestbook)
+	rootCmd.AddCommand(httpecho.CmdHTTPEcho)
```

Comment:
you have a simple echo here , maybe that is just enough

https://github.com/kubernetes/kubernetes/blob/04d87a4b6e72336ee9afb1e5b477223c96a8fcbb/test/images/agnhost/netexec/netexec.go#L85